src/pages/Lectures/LecturesList.js

```diff
@@ -102,9 +103,18 @@ const LecturesList = () => {
             </Row>
           </>
         )
```

Comment:
```suggestion
        ) : (
```
quando fizer ternários, colocar assim fica mais legível